pkgs/build-support/go/module.nix

```
@@ -83,12 +83,16 @@ let
     inherit (args) src;
     inherit (go) GOOS GOARCH;
 
+    # The following inheritence behavior is not trivial to expect, and some may
+    # argue it's not ideal. Changing it may break vendor hashes in Nixpkgs and
+    # out in the wild. In anycase, it's documented in:
+    # doc/languages-frameworks/go.section.md
     prePatch = args.prePatch or "";
     patches = args.patches or [];
     patchFlags = args.patchFlags or [];
     postPatch = args.postPatch or "";
     preBuild = args.preBuild or "";
```

```suggestion
    preBuild = args.preBuild or args.modPostBuild or "";
```
Maybe we should add the parity attribute for preBuild to be a bit consistent

---

> Maybe we should add the parity attribute for preBuild to be a bit consistent

I'm not sure I understand. Perhaps you meant to also respect `args.modPreBuild`? and not `args.modPostBuild`? When setting `preBuild`..

---

Friendly ping @SuperSandro2000 .

---

Yeah, exactly. Forgot to change that when copying